h2d/Console.hx

```diff
@@ -423,9 +423,8 @@ class Console #if !macro extends h2d.Object #end {
 		var scene = ctx.scene;
 		if( scene != null ) {
 			x = 0;
-			y = scene.height - height;
-			width = scene.width;
-			tf.maxWidth = width;

```

Comment:
I removed this line because setting `tf.maxWidth` makes `TextInput` become multiline, is that intented or should I open an issue?